glide.lock

```diff
@@ -13,8 +15,12 @@ imports:
   version: 9a6736ed45b44bf3835afeebb3034b57ed329f3e
   subpackages:
   - go-bindata-assetfs
+- name: github.com/fabric8-services/fabric8-auth
+  version: f67317193c3108895f853f80f9330c1a65b11004
+  subpackages:
+  - design
 - name: github.com/fabric8-services/fabric8-wit
-  version: 58db841e76891f1251bdec86908c61360f11d375
+  version: ff1746954f6c1a89dc2c35b72731e19b63d842d2
```

Comment:
had to update this because of dependencies that got updated as part of https://github.com/fabric8-services/fabric8-auth/commit/468d4e6d7bb84af300d28a432293bcc758b4e708